pre_grey_rgb2D.py

```diff
@@ -10,85 +13,209 @@
 import argparse
 
 # set up the parser
-parser = argparse.ArgumentParser(description='preprocess grey and RGB images')
-parser.add_argument('-i', '--img_path', type=str, default='data/MedSAMDemo_2D/train/images', help='path to the images')
-parser.add_argument('-gt', '--gt_path', type=str, default='data/MedSAMDemo_2D/train/labels', help='path to the ground truth (gt)')
-parser.add_argument('-o', '--npz_path', type=str, default='data/demo2D', help='path to save the npz files')
-parser.add_argument('--data_name', type=str, default='demo2d', help='dataset name; used to name the final npz file, e.g., demo2d.npz')
-parser.add_argument('--image_size', type=int, default=256, help='image size')
-parser.add_argument('--img_name_suffix', type=str, default='.png', help='image name suffix')
-parser.add_argument('--label_id', type=int, default=255, help='label id')
-parser.add_argument('--model_type', type=str, default='vit_b', help='model type')
-parser.add_argument('--checkpoint', type=str, default='work_dir/SAM/sam_vit_b_01ec64.pth', help='checkpoint')
-parser.add_argument('--device', type=str, default='cuda:0', help='device')
-parser.add_argument('--seed', type=int, default=2023, help='random seed')
-args = parser.parse_args()
+parser = argparse.ArgumentParser(description="preprocess grey and RGB images")
 
-names = sorted(os.listdir(args.gt_path))
-save_path = args.npz_path + '_' + args.model_type
-os.makedirs(save_path, exist_ok=True)
-print('image number:', len(names))
-#%% set up the model
-sam_model = sam_model_registry[args.model_type](checkpoint=args.checkpoint).to(args.device)
+# add arguments to the parser
+parser.add_argument(
+    "-i",
+    "--img_path",
+    type=str,
+    default=None,
```

Comment:
the default settings will be used in the 2D tutorial. thus, please keep the default setting:)

Reply:
I have restored all the default values for `--img_path`,`--gt_path` and `--npz_path`